File: front/src/app/models/order.ts
import {OrderProduct} from "./order-product";

export class Order {
  orderId: number;
  firstName: string;
  lastName: string;
  email: string;
  city: string;
  comment: string;
  subtotal: number;
  total: number;
  shippingMethod: string;
  paymentMethod: string;
  orderProducts: OrderProduct[] = [];
}

File: admin/src/main/java/com/vkuzmenko/pizza/admin/dto/Order.java
package com.vkuzmenko.pizza.admin.dto;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

@Data
public class Order {
    private long orderId;
    private String firstName;
    private String lastName;
    private String email;
    private String comment;
    private String orderDate;
    private Integer subtotal;
    private Integer total;
    private String shippingMethod;
    private String paymentMethod;
    private List<OrderProduct> orderProducts = new ArrayList<>();
}
